### PR TITLE
Rover: 4.3 navigation tuning advice

### PR DESCRIPTION
**Pull request by rmackay9**
This moves the existing navigation page (for 4.2 and earlier) to a new page and replaces the existing page with updated advice for 4.3's S-Curves and Position Controller based navigation

I've tested this on my local machine and it seems OK.

### CODE REVIEW

File: rover/source/docs/rover-tuning-navigation.rst [hunk 1]
```diff
@@ -4,38 +4,81 @@
...
 
-Recommended steps for tuning the L1 controller:
+S-Curves are used to plan a smooth path that brings the vehicle close to each waypoint while not exceeding speed or acceleration limits.  The resulting path includes continuous position and 2D velocity targets.
+
+Factors affecting the path include:
+
+- The vehicle will cut the corners but will always try to pass within WP_RADIUS or TURN_RADIUS (whichever is larger) of each waypoint
+- The vehicle will cut the corners more at higher speeds
+- The vehicle will cut the corners more if the maximum accelerations are reduced (e.g. the lowest of ATC_ACCEL_MAX, ATC_DECEL_MAX and TURN_MAX_G * 9.81)
+- The vehicle will slowdown in the corners if necessary to pass within WP_RADIUS of the waypoint without exceeding the maximum accelerations (e.g. the lowest of ATC_ACCEL_MAX, ATC_DECEL_MAX and TURN_MAX_G * 9.81)
```
Comment (yuri-rage) [inline rover/source/docs/rover-tuning-navigation.rst:24]: missing a space in "slow down"

Comment (rmackay9) [reply rover/source/docs/rover-tuning-navigation.rst:24]: fixed, txs!

File: rover/source/docs/rover-tuning-navigation.rst [hunk 2]
```diff
@@ -4,38 +4,81 @@
...
+- Adjust these parameters to improve the vehicle's tracking along the lines
+
+  - set :ref:`PSC_VEL_I<PSC_VEL_I>` and :ref:`PSC_VEL_D<PSC_VEL_D>` to zero
+  - increase :ref:`PSC_VEL_P<PSC_VEL_P>` until the vehicle tracks the corners nicely without oscillating in the straights
+  - increase :ref:`PSC_VEL_D<PSC_VEL_D>` enough to make the corners more responsive but without introducing oscillations in the straights.  This should normally be not more than 10% of :ref:`PSC_VEL_P<PSC_VEL_P>`
+  - set :ref:`PSC_VEL_I<PSC_VEL_I>` to 20% of :ref:`PSC_VEL_P<PSC_VEL_P>`
+
+- These values normally do not need to be changed
+
+  - :ref:`PSC_POS_P<PSC_POS_P>` converts the position error into a desired velocity.  Higher values will lead to the vehicel trying to drive back to the line more quickly but if raised too high may lead to oscillation
```
[the remark this answers is not in the document]
Comment (rmackay9) [reply rover/source/docs/rover-tuning-navigation.rst:50]: fixed, thanks!

**Comment by rmackay9** [comment]
Updated with fixes after peer review, thanks!

**Comment by Hwurzburg** [comment]
very nice....will try this weekend...merging

**Pull request merged**